integration_tests/test_aggregator_settlement.py

```diff
@@ -0,0 +1,174 @@
+# pylint: disable=too-many-instance-attributes, broad-except
+import logging
+import traceback
+import random
+
+from gsy_framework.constants_limits import DATE_TIME_FORMAT
+from pendulum import from_format
+
+from gsy_e_sdk.clients.redis_asset_client import RedisAssetClient
+from integration_tests.test_aggregator_base import TestAggregatorBase
+
+
+class SettlementAggregator(TestAggregatorBase):
+    """Aggregator class to test the connections to settlement
+     markets:
+     Done by asserting whether bids/offers are successfully
+     posted on these markets."""
+
+    def __init__(self, *args, **kwargs):
+
+        self.base_forecast_load = 10
+        self.base_measurement_load = self._calculate_random_deviated_energy(
+            self.base_forecast_load, 5)
+        self.base_forecast_pv = 10
+        self.base_measurement_pv = self._calculate_random_deviated_energy(
+            self.base_forecast_pv, 5)
+
+        self._has_tested_settlement_posts = False
+        super().__init__(*args, **kwargs)
+
+        # self._set_first_measurement_forecast()
+
+    def _setup(self):
+        load_client = RedisAssetClient("forecast-measurement-load", pubsub_thread=self.pubsub)
+        load_client.select_aggregator(self.aggregator_uuid)
+        self.load_uuid = load_client.area_uuid
+
+        pv_client = RedisAssetClient("forecast-measurement-pv", pubsub_thread=self.pubsub)
+        pv_client.select_aggregator(self.aggregator_uuid)
+        self.pv_uuid = pv_client.area_uuid
+
+    def _set_first_measurement_forecast(self):
+        self.add_to_batch_commands.set_energy_forecast(
+            asset_uuid=self.load_uuid,
+            energy_forecast_kWh={"2022-01-12T01:00": self.base_forecast_load}) \
+            .set_energy_forecast(
+            asset_uuid=self.pv_uuid,
+            energy_forecast_kWh={"2022-01-12T01:00": self.base_forecast_pv}) \
+            .set_energy_measurement(
+            asset_uuid=self.load_uuid,
+            energy_measurement_kWh={"2022-01-12T00:00": self.base_measurement_load}) \
+            .set_energy_measurement(
+            asset_uuid=self.pv_uuid,
+            energy_measurement_kWh={"2022-01-12T00:00": self.base_measurement_pv})
+
+        self.send_batch_commands()
```

Comment:
Please remove as it is not used (also the commented call of this method). Thanks!